js/index.js

```diff
@@ -0,0 +1,60 @@
+
+var user_selection = prompt("What Calculator mode do you want to use? Basic, Advanced, BMI or Trip Calculator?");
+
+// Prompt Menu
+
+if (user_selection == "Basic"){
```

Comment:
This works but is very limited. Only works if the user types the word "Basic" exactly. I would consider shortening it down to (b) or (1) or something, to mitigate the risk of invalid input.